cmd/monaco/integration_test_utils.go

```diff
@@ -163,14 +164,15 @@ func cleanupIntegrationTest(t *testing.T, envFile, suffix string, integrationTes
 //
 // <original name>_<current timestamp><defined suffix>
 // e.g. my-config_1605258980000_Suffix
-func RunIntegrationWithCleanup(t *testing.T, configFolder, envFile, suffixTest string, testFunc func(fileReader util.FileReader)) {
+
+func RunIntegrationWithCleanup(t *testing.T, configFolder, envFile, suffixTest string, testFunc func(fileManager files.FileManager)) {
 
 	suffix := getTimestamp() + suffixTest
 	transformers := []func(string) string{getTransformerFunc(suffix)}
 
-	var integrationTestReader, err = util.NewInMemoryFileReader(configFolder, transformers)
-	assert.NilError(t, err)
-
-	testFunc(integrationTestReader)
-	cleanupIntegrationTest(t, envFile, suffix, integrationTestReader)
+	//var integrationTestReader, err = files.NewInMemoryFileManager(configFolder, transformers)
+	var fileManager = files.NewInMemoryFileManager()
+	RewriteConfigNames(configFolder, fileManager, transformers)
```

Comment:
you have dropped checking for errors, but `RewriteConfigNames` may still return an error